plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/InstallManagedCloudSdkStartup.java

```diff
@@ -0,0 +1,28 @@
+package com.google.cloud.tools.eclipse.sdk.ui;
```

Comment:
copyright

Reply:
Hmmm… this should never happen any more.  Ahh! Maybe it's because I created the file from the PDE wizard.

Reply:
Problem solved: this happens as I had "Generate comments" unchecked in the _New Java Class_ wizard.
<img width="424" alt="screen shot 2018-02-14 at 1 28 24 pm" src="https://user-images.githubusercontent.com/202851/36221090-f889e648-118a-11e8-8ac9-bc741ceae0ed.PNG">